content/community/changelog/app-nuget/v4/whats-new/_index.en.md

```diff
@@ -4,6 +4,13 @@ description: Overview of changes introduced in version 4.
 toc: true
 ---
 
+## 4.20.0 (2021-11-15) - Added support for custom redirect URL when exiting an app
+
+The URL to an app can now contain a query parameter (returnUrl) which, if validated (valid URI and host name),
+can be used to redirect the user to that URL when the user clicks on the exit icon in an Altinn 3 app. 
```

Comment:
How about adding: "Read more about the new functionality here" and linking it to the doc you've written?

Reply:
Good suggestion, will fix